x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/control_panel/index.test.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { render } from '@testing-library/react';
+import { shallow } from 'enzyme';
```

Comment:
Maybe consider using only one testing library per test module and not have a mix of the two.

Reply:
I will see how to resolve this. Enzyme has shallow, render doesn't. But with shallow you can't do the clicking. I will follow up next on this. Will create Debt issue.